src/web/web-service/ot_client.cpp

```diff
@@ -0,0 +1,205 @@
+#include "ot_client.hpp"
+
+#include <openthread/platform/toolchain.h>
+
+#include <errno.h>
+#include <inttypes.h>
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <sys/un.h>
+#include <unistd.h>
+
+#include "platform-posix.h"
+#include "common/code_utils.hpp"
+#include "common/logging.hpp"
+
+namespace ot {
+
+Client::Client(void)
```

Comment:
Shall we make `OPENTHREAD_POSIX_APP_SOCKET_NAME` and `timeout` configurable in constructor and make the current values default parameter?